resources/server/players/player.service.ts

```diff
@@ -201,7 +201,7 @@ class _PlayerService {
    **/
   handleUnloadPlayerEvent(src: number) {
     this.deletePlayerFromMaps(src);
-
+    emitNet(PhoneEvents.PLAYER_UNLOADED, src, false);
```

Comment:
Why are you passing an argument here? According to the `PLAYER_UNLOADED`  event, it does not have any parameters.

Reply:
Leftover param from changing the event to be used when unloading, rather than toggling the state.

Reply:
I believe we talked about aiming towards a removal of `ON_INIT`